### DISCUSSION TITLE
How long does it usually take for the a new version to get to Maven?

### DISCUSSION THREAD

**Discussion by sebapian** [google/filament#7178] [Q&A] [answered]
ReadME states that v1.42.2 has been release on Android, but I do not see any updates on Maven. Is there usually a time buffer?

**Comment by romainguy** [comment]
Probably just an issue when pushing to Maven Central.

@bejado @poweifeng

**Comment by bejado** [comment, marked as answer]
1.42.2 should be live on Maven now.

**Comment by sebapian** [comment]
Working now, thank you.
